modules/ROOT/pages/tutorial/tutorial-sso-configuration.adoc

```diff
@@ -300,6 +320,18 @@ dbms.security.oidc.azure.authorization.group_to_role_mapping= "managers" = admin
 dbms.security.oidc.azure.claims.groups=roles
 ----
 
+[role=label--new-5.24]
+. (Optional). If you want to mandate that users exist in the database in order to authenticate and authorize, you can use xref:authentication-authorization/auth-providers.adoc[Auth Providers] to achieve this. Set the `dbms.security.require_local_user` configuration setting to `true` in the _neo4j.conf_ file to enable this mode. For example to create a user `jake` who can authenticate and authorize using Entra, you can use the following Cypher query:
+[source, cypher, role=noplay]
+----
+CREATE USER jake
+SET HOME DATABASE 'jakesHomeDb'
+SET AUTH 'oidc-okta' {SET ID 'jakesUniqueEntraUserId'} // this must match the claim that you configured via dbms.security.oidc.azure.claims.username
```

Comment:
here the provider is named `oidc-okta`, the id has `Entra` and the comment `oidc.azure`, feels inconsistent :P

Reply:
very!  Thanks, changed.